routes/movies.test.js

```diff
@@ -31,72 +31,175 @@ describe("/movies routes", () => {
-      expect(false).toEqual(true);
+      movieData.create.mockResolvedValue({_id:"890", title:"One Day"});
+
+      const res = await request(server).post("/movies/890");
+
+      expect(res != null && typeof res === 'object').toEqual(true);
     });
     it("should return an error message if body is missing title", async () => {
-      expect(false).toEqual(true);
-      // expect status code == 400
+      movieData.create.mockResolvedValue({error: "Movies must have a title."});
+
+      const res = await request(server).post("/movies/890");
+
+      expect(res.error).toBeDefined();
     });
     it("should return an error message if movie fails to be created", async () => {
-      expect(false).toEqual(true);
-      // expect status code == 400
+      movieData.create.mockResolvedValue({error: "Could not create object"});
+      const res = await request(server).post("/movies/890");
+      expect(res.error).toBeDefined();
+    });
+  });
+
+  describe("POST /:movieId/comment/", () =>{
+
+    it("should return the new comment on success", async () => {
+      movieData.createComment.mockResolvedValue({_id:"890", comment:"One Day"});
+
+      const res = await request(server).post("/movies/890/comment");
+
+      expect(res != null && typeof res === 'object').toEqual(true);
+    });
+    /*
+    it("should return an error message if body is missing title", async () => {
+      movieData.create.mockResolvedValue({error: "Movies must have a title."});
+
+      const res = await request(server).post("/movies/890/comment");
+
+      expect(res.error).toBeDefined();
+    });
+    */
+
+    it("should return an error message if comment fails to be created", async () => {
+      movieData.create.mockResolvedValue({error: "Could not create object"});
+      const res = await request(server).post("/movies/890/comment");
+      expect(res.error).toBeDefined();
     });
   });
 
   describe("PUT /:id", () =>{
     it("should return the updated movie on success", async () => {
-      expect(false).toEqual(true);
+      movieData.updateById.mockResolvedValue({_id:"890", title:"One Day"});
+      const res = await request(server).put("/movies/890");
+      expect(res.body != null && typeof res.body === 'object').toEqual(true);
     });
     it("should return an error message if movie fails to be updated", async () => {
-      expect(false).toEqual(true);
+      movieData.updateById.mockResolvedValue({error: "Could not update object"});
+      const res = await request(server).put("/movies/890");
+      expect(res.error).toBeDefined();
+    });
+  });
+
+  describe("PUT /:movieId/comments/:id", () =>{
+    it("should return the updated comment on success", async () => {
+      movieData.updateCommentById.mockResolvedValue({_id:"890", comment:"Test Comment"});
+      const res = await request(server).put("/movies/890/comments/27");
+      expect(res.body != null && typeof res.body === 'object').toEqual(true);
```

Comment:
While I appreciate the brevity, with unit testing it's valuable to write a separate assertion for each condition we're testing for. Tests often serve as documentation, so it's best that they are very explicit.